apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -155,6 +155,10 @@ type FeatureGates struct {
 	// ServiceMonitor defines whether the ServiceMonitor CRD exists in the cluster.
 	// This CRD is part of prometheus-operator.
 	ServiceMonitor bool `json:"serviceMonitor,omitempty"`
+
+	// EnableAlerts defines if  PrometheusRules will be created for alerting.
+	// This CRD is part of prometheus-operator.
+	EnableAlerts bool `json:"enableAlerts,omitempty"`
```

Comment:
To be consistent with the `ServiceMonitor` feature gate above, maybe call this one `PrometheusRule`?

Calling it `EnableAlerts` sounds like it will actually enable alerts, however this feature gate only specifies if the `PrometheusRule` CRD exists in the cluster, and does not create any alerts by itself.

However, thinking more of it, if the `ServiceMonitor` CR exists then also the `PrometheusRule` CR exists. Maybe we should merge both flags into one `PrometheusOperator` flag or something? What are your thoughts?

Reply:
Not sure about merge it in only one, but on the other side, I cannot think in a use case where you want to enable serviceMonitors and disable Rules.

Reply:
I'd only merge this feature gate, not the flag in the CRD. So this featuregate basically only says "prometheus operator is available or not", and in the TempoStack CR users can enable or disable metrics and alerts independently (well, if you enable alerts but disable metrics than the alerts won' work...hmm).